addon/lib/connect.js

```diff
@@ -0,0 +1,46 @@
+import Ember from 'ember';
+import reduxStore from '../services/redux-store';
+
+const {
+  computed,
+  defineProperty,
+  inject: { service }
+} = Ember;
+
+export default function connect(mapStateToComputed, mapDispatchToActions, storeService) {
```

Comment:
Mmm...right...it **can** be a function...or an objective...hmm. Finding the naming a little opaque, but don't have a great alternative suggestion yet.